src/color/color_utils.js

```diff
@@ -11,22 +11,24 @@ define(function(require) {
   /**
    * For a color expressed as an HSBA array, return the corresponding RGBA value
    * 
-   * @param {Array} hsba An 'array' object that represents a list of
-   *                          HSB colors on a scale of 0-255
+   * @param {Array} hsba An 'array' object that represents a list of HSB colors
+   * @param {Array} maxes An 'array' object that represents the HSB range maxes
    * @return {Array} an array of RGBA values, on a scale of 0-255
    */
-  p5.ColorUtils.hsbaToRGBA = function(hsba) {
+  p5.ColorUtils.hsbaToRGBA = function(hsba, maxes) {
     var h = hsba[0];
     var s = hsba[1];
     var v = hsba[2];
-    h /= 255;
-    s /= 255;
-    v /= 255;
+    var a = hsba[3] || maxes[3];
+    h /= maxes[0];
+    s /= maxes[1];
+    v /= maxes[2];
+    a /= maxes[3];
```

Comment:
Again, I didn't write this; I just followed the established file. If we need to discuss styles, it should be in an actual issue and not in line notes.